spyder/plugins/editor/lsp/manager.py

```diff
@@ -36,6 +36,9 @@ class LSPManager(QObject):
     CONF_SECTION = 'lsp-server'
     LOCALHOST = ['127.0.0.1', 'localhost']
 
+    # For testing
+    sig_initialize = Signal(dict, str)
```

Comment:
On Python 2 are there any issues on using `str` vs `unicode` ? Should we just use `object` ?

Reply:
No problem here, because the `str` is the language name we created an LSP client for. And programming languages don't have unicode names.